tests/integration/remote/util.py

```diff
@@ -105,8 +107,12 @@ def do_ocean_tx_and_handle_gotchas(ocean, alice_wallet):
 
     print("Call create() from data NFT, and wait for it to complete...")
     try:
+        tx_dict = {"from": alice_wallet}
```

Comment:
I suggest creating the entire tx dict at once since there is no conditional here.

I.e.
```
data_nft = ocean.data_nft_factory.create(
            DataNFTArguments(symbol, symbol), {"from": alice_wallet, "gas_price": gas_strategy.max_gas_price}
        )
```

Reply:
Ok thank you